addons/account/static/src/js/no_content_help_widget.js

```diff
@@ -0,0 +1,62 @@
+odoo.define('web.noContentHelpWidget', function (require) {
+"use strict";
+var BasicRenderer = require('web.BasicRenderer');
+var Widget = require('web.Widget');
+
+var noContentHelpWidget = Widget.extend({
```

Comment:
this widget should be defined in web module, in place of the current implementation of the no_content help